dev/integration_tests/flavors/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -539,7 +542,10 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = "Runner/Info-Free.plist";
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
```

Comment:
Seems like these changes have been formatted by Xcode when I opened project in it. Should it be reverted or it's ok ?